lib/db/mysql/index.js

```diff
@@ -104,6 +104,14 @@ const QUERY_CLIENT_REGISTER =
   'INSERT INTO clients ' +
   '(id, name, imageUri, secret, redirectUri, whitelisted, canGrant) ' +
   'VALUES (?, ?, ?, ?, ?, ?, ?);';
+const QUERY_CLIENT_DEVELOPER_INSERT =
+  'INSERT INTO clientDevelopers ' +
+  '(userId, clientId) ' +
+  'VALUES (?, ?);';
+const QUERY_DEVELOPER_INSERT =
+  'INSERT INTO developers ' +
+  '(email) ' +
```

Comment:
TODO: add developerId here, part of https://github.com/mozilla/fxa-oauth-server/pull/222/files#r26528695